components/typography/TextLink.tsx

```diff
@@ -35,7 +35,7 @@ export default function TextLink({
       className={classNames}
       id={id}
       data-testid='TextLink-href'
-    >
+      legacyBehavior>
```

Comment:
Why we added legacyBehavior here?

Reply:
i used "npx @next/codemod new-link" to fix link components in every file which contained link component 

https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor